src/edu/csus/ecs/pc2/core/InternalController.java

```diff
@@ -3251,19 +3196,28 @@ private Profile getCurrentProfile() {
 
     private void handleCommandLineOptions() {
 
-        if (parseArguments.isOptPresent("--help")) {
+        if (parseArguments.isOptPresent(AppConstants.HELP_OPTION_STRING)) {
             // -F is the ParseArguements internal option to pre-load command line options from a file
 
             String[] usage = { //
-                    "Usage: Starter [--help] [-F filename] [--server] [--first] [--login <login>] [--password <pass>] " + //
-                    "[" + LOAD_OPTION_STRING + " <dir>|<file> ] " + //
-                    "[--skipini] " + //
-                    "[" + NOLOGGING_OPTION_STRING + "] ", //
-                    "[" + INI_FILENAME_OPTION_STRING + " filename] [" + //
-                    CONTEST_PASSWORD_OPTION + " <pass>] [" + NO_GUI_OPTION_STRING + "] "+ //
-                    "[" + NOSTANDINGS_OPTION_STRING + "] "+ //
-                    "[" + REMOTE_SERVER_OPTION_STRING + " <remoteHostname> --proxyme]" + //
-                    "[" + MAIN_UI_OPTION + " classname]", // 
+                    "Usage: Starter " + //
+                    "[" + AppConstants.HELP_OPTION_STRING + "] " + //
+                    "[" + AppConstants.FILE_OPTION_STRING + " filename] " + //
+                    "[" + AppConstants.SERVER_OPTION_STRING + "] " + //
+                    "[" + AppConstants.FIRST_SERVER_OPTION_STRING + "] " + //
+                    "[" + AppConstants.LOGIN_OPTION_STRING + " <login> " + //
+                    "[" + AppConstants.PASSWORD_OPTION_STRING + " <pass> ", //
+                    "           " + 
```

Comment:
What is this series of blanks intended for?  It's not present in the original output.  It seems like it would tend to make the usage string scroll unnecessarily far to the right on the screen (and it's already pretty long...)

Reply:
Each string in the usage array is printed on its own line.  The "     " is meant to indent the following line so it offset from the left of the screen and lines up under the other options.  The comma on the previous line can be changed to a plus to make everything on the same line if that is desired (and, of course, remove the "      ").